### PR TITLE
Upgrate Login, Register pages

### PR DESCRIPTION
**Pull request by AzamatG17**
Add text about company, Remove start page

### CODE REVIEW

File: Inflow.Mobile/Inflow.Mobile/Inflow.Mobile/Views/LoginPage.xaml [hunk 1]
```diff
@@ -7,78 +7,13 @@
              mc:Ignorable="d"
              x:Class="Inflow.Mobile.Views.LoginPage"
              Shell.NavBarIsVisible="False" FlowDirection="LeftToRight" >
-
+
```
Comment (Mirazyzz) [inline Inflow.Mobile/Inflow.Mobile/Inflow.Mobile/Views/LoginPage.xaml:10]: Remove redundant empty line.

**Review by Mirazyzz — CHANGES_REQUESTED** [review]
Please, install following extension for your visual studio and format xaml code, then push it again.
https://marketplace.visualstudio.com/items?itemName=TeamXavalon.XAMLStyler2022

File: Inflow.Mobile/Inflow.Mobile/Inflow.Mobile/Views/LoginPage.xaml (hunk 1 above)
Comment (AzamatG17) [reply Inflow.Mobile/Inflow.Mobile/Inflow.Mobile/Views/LoginPage.xaml]: I corrected everything according to the Xaml format

**Review by Mirazyzz — APPROVED** [review]
LGTM.

**Review by khojianvar — APPROVED** [review]

**Review by diyorzz — APPROVED** [review]

**Review by SharifovDeveloper — APPROVED** [review]

**Review by SharafiddinMukhtorov — APPROVED** [review]

**Pull request merged**